art/api/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
-# Create your models here.
+
+class AssetCategory(models.Model):
+    """ Stores all asset categories """
+    name = models.CharField(max_length=40, null=False)
```

Comment:
Can we use something like `category_name`? `name` doesn't sound descriptive enough :)

Reply:
I see your point. I will change this.